float64 vr
float64 u0
float64 i0
float64 wv
float64 wi
float64 i_max
float64 i_min
---
float64 i
